src/sidebar/components/selection-tabs.js

```diff
@@ -108,6 +111,7 @@ function SelectionTabs({ isLoading, settings, session }) {
         className={classnames('selection-tabs', {
           'selection-tabs--theme-clean': isThemeClean,
         })}
+        role="tablist"
```

Comment:
It would be cool if, in the future, `Tab` components could render as `button` elements with a `role="tab"`, as that is a little more standard, but this is definitely an improvement for now!

Reply:
Given that we're already setting a custom role, would using `<button>` make any difference here? I suppose it might if one could find a browser that doesn't understand `role="tab"`.